build.gradle

```diff
@@ -47,6 +47,15 @@ idea.project {
   languageLevel = JavaVersion.VERSION_1_7
 }
 
+// Gobblin build customization
+if (!project.hasProperty('gobblinFlavor')) {
```

Comment:
It seems there are a few distinct supported flavors, can we document them somewhere to be explicit about what version does what?